HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs

```diff
@@ -51,6 +51,21 @@ public ActionResult getPublishedFeedbacks()
             }
         }
 
+        [HttpPut]
+        public ActionResult PublishFeedback(FeedbackDTO feedbackDTO)
+        {
+            try
+            {
+                Feedback feedback = FeedbackAdapter.FeedbackDTOToFeedback(feedbackDTO);
+                _feedbackService.PublishFeedback(feedback);
```

Comment:
Another thing that I just realized is that the way this method is implemented in the FeedbackService isn't actually very good - it can currently be used to perform any update, not specifically publish. I feel like this branch might be a good place to fix that.